docs/scos/dev/upgradability-services/upgradability-guidelines/private-api-is-used-on-the-project-level.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Private API is used on the project level
+description: Reference information for evaluator and upgrader tools.
+last_updated: Mar 23, 2022
+template: concept-topic-template
+---
+
+Modules have public and private APIs. While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility is not guaranteed in the private API. For example, if you use a core method on the project level, and it is updated or removed, it can cause unexpected issues during updates.
```

Comment:
```suggestion
Modules have public and private APIs. While public API updates always support backward compatibility, private API updates can break backward compatibility. So, backward compatibility is not guaranteed in the private API. For example, if you use a core method from the Private API on the project level, and it is updated or removed, it can cause unexpected issues during updates.
```